src/com/company/Professor.java

```diff
@@ -0,0 +1,29 @@
+package com.company;
+
+
+public class Professor extends UniversityPerson {
+
+    public Professor(){
+    numbOfUP++;
```

Comment:
what is it ?

Reply:
Just for calculate numbers of university persons.

Reply:
> what is it ?
Just for calculate numbers of university persons.

Reply:
name is not clear